lldb/source/Plugins/Process/elf-core/RegisterContextPOSIXCore_arm64.cpp

```diff
@@ -67,6 +70,21 @@ RegisterContextCorePOSIX_arm64::RegisterContextCorePOSIX_arm64(
     : RegisterContextPOSIX_arm64(thread, std::move(register_info)) {
   ::memset(&m_sme_pseudo_regs, 0, sizeof(m_sme_pseudo_regs));
 
+  ProcessElfCore *process =
+      static_cast<ProcessElfCore *>(thread.GetProcess().get());
```

Comment:
I don't think you need to do this right now, but it would be nice if we could move away from `static_cast` and start using the llvm RTTI support with `llvm::cast` instead...